notebooks/224-stable-diffusion-text-to-image/224-stable-diffusion-text-to-image.ipynb

```diff
@@ -0,0 +1,1209 @@
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "!pip install -r requirements.txt"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Create Pytorch Models pipeline\n",
+    "StableDiffusionPipeline is an end-to-end inference pipeline that you can use to generate images from text with just a few lines of code.\n",
+    "\n",
+    "First, we load the pre-trained weights of all components of the model.\n",
+    "\n",
+    "In addition to the model id `CompVis/stable-diffusion-v1-4`, we're also passing a `use_auth_token` to the from_pretrained method `use_auth_token` to verify that you have indeed accepted the model's license."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 134,
+   "metadata": {
+    "colab": {
+     "base_uri": "https://localhost:8080/"
+    },
+    "id": "8NS0mazM0icN",
+    "outputId": "b98b31ca-65ca-4bb0-fad2-2599c668ccad"
+   },
+   "outputs": [
+    {
+     "data": {
+      "application/vnd.jupyter.widget-view+json": {
+       "model_id": "fe1a770f02cb44ee9c75d1ad65739816",
+       "version_major": 2,
+       "version_minor": 0
+      },
+      "text/plain": [
+       "Fetching 15 files:   0%|          | 0/15 [00:00<?, ?it/s]"
+      ]
+     },
+     "metadata": {},
+     "output_type": "display_data"
+    },
+    {
+     "name": "stderr",
+     "output_type": "stream",
+     "text": [
+      "ftfy or spacy is not installed using BERT BasicTokenizer instead of ftfy.\n"
+     ]
+    }
+   ],
+   "source": [
+    "from diffusers import StableDiffusionPipeline\n",
+    "\n",
+    "pipe = StableDiffusionPipeline.from_pretrained(\"prompthero/midjourney-v4-diffusion\")"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Convert models to OpenVINO Intermediate representation format\n",
+    "\n",
+    "OpenVINO supports PyTorch\\* through export to the ONNX\\* format. We will use `torch.onnx.export` function for obtaining ONNX, \n",
+    "you can find more info about it in [PyTorch documentation](https://pytorch.org/docs/stable/onnx.html). \n",
+    "We need provide model object, input data for model tracing and path for model saving. \n",
+    "Optioanally, we can provide target onnx opset for conversion and other parameters specified in documentation (e.g. input and output names or dynamic shapes).\n",
+    "\n",
+    "While ONNX models are directly supported by OpenVINO™, it can be useful to convert them to IR format to take advantage of advanced OpenVINO optimization tools and features. `mo.convert` function can be used for converting model using OpenVINO Model Optimizer capabilities. It returns of instance OpenVINO Model class, which is ready to use in python interface and can be serialized to IR for next usages.\n",
```

Comment:
```suggestion
    "While ONNX models are directly supported by OpenVINO runtime, it can be useful to convert them to IR format to take advantage of advanced OpenVINO optimization tools and features. The `mo.convert` Python function can be used to convert a model to OpenVINO IR using Model Optimizer. This returns an instance of OpenVINO Model class, which is ready to use in Python interface but can also be serialized to IR format for future use.\n",
```